monai/handlers/ignite_loss_metric.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from collections.abc import Callable
+
+from monai.handlers.ignite_metric import IgniteMetric
+from monai.utils import MetricReduction
+from monai.metrics import LossMetric
+
+from torch.nn.modules.loss import _Loss
+
+class IgniteLossMetric(IgniteMetric):
```

Comment:
Agree with Wenqi, we can update this class name to `LossMetricHandler` which may make it clearer.